CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Removed
 
+## [1.13.1] - 2019-06-18
+
+### Added
+- Transaction: added new public fields
```

Comment:
Pls specify which type of transaction (we have Payment and Smart, so Smart in this case) is modified, and which fields are added